nimble/host/src/ble_gap.c

```diff
@@ -6062,6 +6063,9 @@ ble_gap_unpair(const ble_addr_t *peer_addr)
 
     ble_hs_pvcy_remove_entry(peer_addr->type,
```

Comment:
looks like return value from  ble_hs_pvcy_remove_entry() is never stored in rc...

Reply:
Thanks for your suggestion. So I have made further changes including above mentioned in this API and will update this PR.